lib/ansible/cli/scripts/ansible_cli_stub.py

```diff
@@ -92,7 +92,7 @@ def error(self, msg, wrap_text=None):
             mycli = getattr(__import__("ansible.cli.%s" % sub, fromlist=[myclass]), myclass)
         except ImportError as e:
             # ImportError members have changed in py3
-            if 'msg' in dir(e):
+            if hasattr(e, 'msg'):
```

Comment:
This is a definitely a sane substitution. This is what initially caught my attention.